components/director/internal/domain/labeldef/resolver.go

```diff
@@ -134,6 +148,11 @@ func (r *Resolver) UpdateLabelDefinition(ctx context.Context, in graphql.LabelDe
 	if err != nil {
 		return nil, err
 	}
+	// TODO: Use LabelDefinitionInput
```

Comment:
Is this still valid?

Reply:
And also why did you move conversion to the top? If there will be transaction error, then there is no need to convert input to actual LabelDefinition.

Reply:
Hmm, right, but I would keep the transaction at the beginning. For now we need to remember about opening it, so it's good to have it unified across all places. When we implement #257 then the problem will be gone. That's why please move back to opening transaction at the beginning of resolver.